### ISSUE TITLE
MicrosoftTeams module supports PS7

### ISSUE THREAD

**Issue by shivtorov** [get-itips/msshells#97]
Microsoft Teams module officially supported with PowerShell 7.

> From: https://www.powershellgallery.com/packages/MicrosoftTeams/
> "Microsoft Teams cmdlets module for Windows PowerShell **and PowerShell Core**"
> 
> From: https://docs.microsoft.com/en-us/microsoftteams/teams-powershell-install
> "Microsoft Teams PowerShell module requires PowerShell 5.1 **or higher**"

By the way, there is an easier way check latest versions of modules:

```PowerShell

$modulesToBeChecked = @'
Name
MicrosoftTeams
ExchangeOnlineManagement
Microsoft.Online.SharePoint.PowerShell
AzureAD
AzureADPreview
Az
WhiteboardAdmin
MicrosoftPowerBIMgmt
Microsoft.PowerApps.Administration.PowerShell
Microsoft.PowerApps.PowerShell
MSCommerce
'@ | ConvertFrom-Csv

# latest versions
$modulesToBeChecked | Find-Module

# latest prelease versions
$modulesToBeChecked | Find-Module -AllowPrerelease
```

**Comment by get-itips** [comment]
Hello @shivtorov how are you? Thanks for your feedback.
I think that sentence has been there for a long time, however, you will also find this sentence in the Docs article
> Microsoft Teams PowerShell module is currently supported for use with PowerShell 5.1 on Windows. 
And from my tests, there are still issues with authentication in Core.

@robdy Mind reviewing what @shivtorov is telling about module check? :)

**Comment by robdy** [comment]
@shivtorov yes, you're right. I initially thought that using web requests is faster, but it's not. Do you want to open a PR for that or you'd prefer us to take care of it?

**Comment by shivtorov** [comment]
@robdy, I don't have any experience with GitHub Actions, so I definitely can't provide any help here.

But you have a working solution. There is no reason to change something that is not broken. Keep it as is 😉

**Comment by robdy** [comment]
> There is no reason to change something

@shivtorov I do not agree, your solution is far more readable 👍  I already opened a PR for that, it'll be merged soon (I hope)
